tests/AbstractDoctrineAnnotationFixerTestCase.php

```diff
@@ -86,7 +86,10 @@ private function withClassDocBlock($comment)
 %s
 class FooClass
 {
-}', $comment, false);
```

Comment:
I'm not sure those cases should be fixed: newlines are within strings only, not around arguments. Also, a newline is now required after opening braces.

If those changes were made by the new behavior introduced in this PR, then you should add test cases to cover this.